security/compliance_operator/co-management/compliance-operator-installation.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 
 Before you can use the Compliance Operator, you must ensure it is deployed in the cluster.
 
+[IMPORTANT]
+====
+Compliance Operator scans will report INCONSISTENT on clusters with `arm-mixarch` (control plane is `arm`) or `amd-mixarch` (control plane is `amd`). For more information, see xref:../co-scans/compliance-operator-supported-profiles.adoc#compliance-operator-supported-profiles[Supported compliance profiles]
```

Comment:
1. Maybe bette to move to running-compliance-scans_compliance-operator-scans part, for example https://92420--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/co-scans/compliance-scans#running-compliance-scans_compliance-operator-scans
2. Better to update to 
Compliance Operator scans will report INCONSISTENT on clusters with multi-architecture compute machines, no matter control plane is based on `aarch64` or `x86`. For more information, see xref:../co-scans/compliance-operator-supported-profiles.adoc#compliance-operator-supported-profiles[Supported compliance profiles]